packages/agoric-cli/src/commands/vaults.js

```diff
@@ -59,6 +77,116 @@ export const makeVaultsCommand = async logger => {
       }
     });
 
+  vaults
+    .command('info')
+    .description('show governance parameters of a vault type')
+    .requiredOption('--manager [N]', 'manager number', s => Number(s), 0)
+    .action(async ({ manager }) => {
+      const { entries, values } = Object;
+      /** @type {import('../lib/format.js').AssetDescriptor[]} */
+      // @ts-expect-error Brand vs. RpcRemote
+      const assets = values(agoricNames.vbankAsset).map(d => ({
+        ...d,
+        petname: d.issuerName,
+      }));
+      const fmtAmt = makeAmountFormatter(assets);
+      console.log(`manager${manager} governance parameters:`);
+      const { governance } = await getGovernanceState(manager);
+      for (const [name, { type, value }] of entries(governance)) {
+        switch (type) {
+          case 'amount':
+            console.log(name, fmtAmt(value));
+            break;
+          case 'ratio':
+            console.log(name, asPercent(value), '%');
+            break;
+          default:
+            console.log(name, type, value);
+        }
+      }
+    });
+
+  vaults
+    .command('proposeParamChange')
+    .description('propose to change a parameter')
+    .option(
+      '--offerId [string]',
+      'id of this offer (optional)',
+      String,
+      `proposeParamChange-${Date.now()}`,
+    )
+    .requiredOption('--manager [N]', 'manager number', s => Number(s), 0)
+    .requiredOption(
+      '--previousOfferId [string]',
+      'offer using psm charter invitation',
+    )
+    .addOption(
+      new Option('--param <param>', 'parameter').makeOptionMandatory().choices([
+        // TODO: amounts: 'DebtLimit'
+        'InterestRate',
+        'LiquidationMargin',
+        'LiquidationPadding',
+        'LiquidationPenalty',
+        'LoanFee',
+      ]),
+    )
+    .requiredOption(
+      '--percent <percent>',
```

Comment:
so this command is a `proposeRatioParamChange`?

or do you want to change the validation when it supports `debtLimit`?